mongodbatlas/config.go

```diff
@@ -23,14 +31,13 @@ type Config struct {
 	RealmBaseURL string
 }
 
-// MongoDBClient client
+// MongoDBClient contains the mongodbatlas clients and configurations
 type MongoDBClient struct {
-	Atlas  *matlasClient.Client
-	Config *Config
+	Atlas    *matlasClient.Client
+	AtlasSDK *atlasSDK.APIClient
```

Comment:
nit - in Atlas CLI we tend to add prefix `v2` to the objects. 
It doesn't mean SDK v2 but it flags that SDK is using v2 API. 
In long term that would also make difference as we going to see v2 only API changes.
```suggestion
	AtlasV2 *atlasSDK.APIClient
```